client/verta/tests/registry/test_model.py

```diff
@@ -132,6 +132,24 @@ def test_description(self, client, created_entities):
         registered_model.set_description(desc)
         assert desc == registered_model.get_description()
 
+    def test_pii(self, client, created_entities):
+        # Test setting on creation
+        registered_model = client.get_or_create_registered_model(pii=True)
+        created_entities.append(registered_model)
+        assert registered_model.get_pii()
```

Comment:
For maximum correctness because in Python, lots of non-`bool` objects can be `True`y and therefore pass an `assert`. We should explicitly check that it's `True`.

```suggestion
        assert registered_model.get_pii() is True
```

Reply:
ah, caught this for the False case but didn't think of it for True